{"name": "Тест на создание фейковых данных", "status": "broken", "statusDetails": {"message": "AttributeError: 'Database' object has no attribute 'test_create_fake_products'", "trace": "@allure.title(\"Тест на создание фейковых данных\")\n    def test_create_fake_data():\n        # Используем методы объекта db для создания фейковых данных\n>       db.test_create_fake_products(count=2)\nE       AttributeError: 'Database' object has no attribute 'test_create_fake_products'\n\napi_test.py:16: AttributeError"}, "start": 1703592363363, "stop": 1703592363363, "uuid": "2d8b52bd-c40a-4e46-a803-641e4b51e0aa", "historyId": "b355e833ec90965f54a795af8b707372", "testCaseId": "b355e833ec90965f54a795af8b707372", "fullName": "api_test#test_create_fake_data", "labels": [{"name": "suite", "value": "api_test"}, {"name": "host", "value": "DESKTOP-TV6CGIP"}, {"name": "thread", "value": "11068-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "api_test"}]}